README.md

```diff
@@ -300,11 +300,16 @@ TorchAO also supports arbitary exponent and mantissa bits for floating point typ
 |  model_type  |  compile  |  fuse_qkv  |  quantize_vae  |  quantization  |   model_memory |   inference_memory |   time  |
```

Comment:
I think we should also edit this note:

> TorchAO also supports arbitary exponent and mantissa bits for floating point types, which provides experimental freedom to find the best settings for your models. Here, we also share results with fp6_e3m2, fp5_e2m2 and fp4_e2m1. We find that fp6 and fp5 quantizations can preserve good generation quality and match the expectation from fp16 precision most of the time. _Note that running compiled inference with these types fails at the moment_.

Reply:
Removed. Compilation works with all datatypes now. One quirk is that fp8 stuff is only supported on H100s, but might be interesting to see if `fpx_weight_only(4, 3)` or `fpx_weight_only(5, 2)` would work on Ampere/Turing.

Reply:
Let's mention this limitation as well?

Reply:
Already mentioned in our current README :)

![image](https://github.com/user-attachments/assets/393f0456-0307-442e-adec-e98eb1745646)